variables.tf

```diff
@@ -160,6 +160,24 @@ variable "enable_nat_gateway" {
   default     = true
 }
 
+variable "vpc_endpoint_service_acceptance_required" {
+  description = "(Required) Whether or not VPC endpoint connection requests to the service must be accepted by the service owner - true or false."
+  type        = bool
+  default     = true
+}
+
+variable "vpc_endpoint_service_allowed_principals" {
+  description = "(Optional) The ARNs of one or more principals allowed to discover the endpoint service."
+  type        = list(string)
+  default     = null
+}
+
+variable "graphdb_enable_private_access" {
```

Comment:
Shouldn't this be `lb_enable_priavate_access` ? after all the private link leads to the network lb and not directly to GDB